src/calendar-heatmap.component.js

```diff
@@ -1394,6 +1407,26 @@ export class CalendarHeatmap extends Component {
       .rangeRound([this.settings.label_padding, this.settings.height])
       .domain(project_labels);
 
+    // Define beginning and end of the day
+    let start_of_day = moment(this.selected.date).startOf('day');
+    let end_of_day = moment(this.selected.date).endOf('day');
+
+    // Filter data down to the selected week
+    let day_data = this.props.data.filter((d) => {
+      return start_of_day <= moment(d.date) && moment(d.date) < end_of_day;
+    });
+    const daySummaries = day_data.flatMap((e) => e.summary);
```

Comment:
do all the elements have `summary`

Reply:
@maneetgoyal sir, the reason author was using `summary` is mainly for the presentation of overall data in the `tooltip` that's why for every type of overview `year, day, month etc` we have `summary`